connectivity/suite.go

```diff
@@ -166,6 +169,29 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 			return check.ResultOK, check.ResultNone
 		})
 
+	// Test L7 HTTP introspection abilities and egress filtering.
+	// The policy allows ambient DNS resolution,
+	ct.NewTest("client-egress-l7").
```

Comment:
Can we also add `echo-ingress-l7` as was pointed out in #217?

Reply:
Okay, sure. Does it make a difference in practice whether an L7 policy is applied using an `endpointSelector` or a `toEndpoints` clause? We should try to avoid redundant tests, but I'm not opposed to having a separate L7 ingress test.

Reply:
Okay, reading through https://github.com/cilium/cilium/issues/15826, it looks like there is indeed a difference. Will add an ingress equivalent as well. :+1:

Reply:
Added `echo-ingress-l7` test.